Themes/default/languages/Login.english.php

```diff
@@ -155,4 +155,4 @@
 $txt['logout_notice'] = 'You are about to be logged out of the forum and continue browsing as a guest!';
 $txt['logout_return'] = 'Stay logged in and return to browsing as a member.';
 
-?>
```

Comment:
Please do not change this trailing new line. Use a better editor that will not auto change it. e.g. Visual Studio Code

Reply:
Well, it's the GitHub web-based editor.. but thanks for the suggestion.